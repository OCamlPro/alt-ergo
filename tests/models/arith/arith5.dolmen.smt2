(set-option :produce-models true)
(set-option :verify-models false)
(set-logic ALL)
(declare-const x Real)
(declare-const y Real)
(declare-const z Real)
; This linear program is bounded.
(assert (= (+ (* 5.0 x) (* 2.0 y) (* (- 0.0 3.0) z)) 2.0))
(assert (= (+ (* (- 0.0 2.0) x) (* 4.0 y)) 0.0))
(assert (= (+ (* 2.0 x) (* 4.0 y) (* 1.0 z)) (- 0.0 1.0)))
(maximize x)
(check-sat)
(get-model)
(get-objectives)
