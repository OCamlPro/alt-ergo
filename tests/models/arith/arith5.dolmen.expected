
unknown
(
  (define-fun x () Real (/ (- 1) 18))
  (define-fun y () Real (/ (- 1) 36))
  (define-fun z () Real (/ (- 7) 9))
)
(objectives 
  (x (/ (- 1) 18))
)